api/v2/api.go

```diff
@@ -721,3 +770,91 @@ func getSwaggerSpec() (*loads.Document, *analysis.Spec, error) {
 	swaggerSpecAnalysisCache = analysis.New(swaggerSpec.Spec())
 	return swaggerSpec, swaggerSpecAnalysisCache, nil
 }
+
+func (api *API) getAlertsFromAlertGroup(receiverFilter *regexp.Regexp, alertFilter func(a *types.Alert, now time.Time) bool, groupIdsFilter func(groupId string) bool) (open_api_models.GettableAlerts, error) {
+	res := open_api_models.GettableAlerts{}
+	routeFilter := api.routeFilter(receiverFilter)
+	alertGroups, allReceivers := api.alertGroups(routeFilter, alertFilter, groupIdsFilter)
+	for _, alertGroup := range alertGroups {
+		for _, alert := range alertGroup.Alerts {
```

Comment:
`getAlerts` check for `ctx.Err()`. Should we do the same here?

Reply:
done